Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -0,0 +1,59 @@
+# How to Configure Oculus Quest in MRTK using the XRSDK pipeline
```

Comment:
Updated to the [sentence case recommendation for headers](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#capitalization) in our docs:

```suggestion
# How to configure Oculus Quest in MRTK using the XRSDK pipeline
```